src/models/reposyncstate.ts

```diff
@@ -310,6 +310,24 @@ export class RepoSyncState extends Model implements RepoSyncStateProperties {
 		return repositories as RepoSyncState[];
 	}
 
+	static async findOneForRepoUrlAndRepoId(repoUrl: string, repoId: string) {
```

Comment:
I didn't get a chance to do this during inno week but we should probably add a test/s for this.

Reply:
Will do 👍